ibis/backends/dask/helpers.py

```diff
@@ -47,7 +47,7 @@ def asseries(cls, value, like=None):
         elif isinstance(value, pd.Series):
             return dd.from_pandas(value, npartitions=1)
         elif like is not None:
-            if isinstance(value, (tuple, list, dict)):
+            if isinstance(value, (tuple, list, dict, np.ndarray)):
```

Comment:
ops.Array executes to a np.ndarray (both before this change, and after). We never ran into this scenario in tests before, so this tweak here is adjusting an existing bug.